src/containers/VariableStep.tsx

```diff
@@ -66,6 +77,20 @@ const VariableStep = ({ number, active, variables, method, setDefaultVariables }
         </div>
       </div>
 
+      <div className="margin-bottom-10">
+        <input
+          className="is-checkradio is-rtl is-info"
+          id="customMode"
+          type="checkbox"
+          name="customMode"
+          checked={customMode}
+          onChange={() => onCustomModeChange(!customMode)}
+        />
+        <label htmlFor="customMode" className="ml-0">
```

Comment:
`is-rtl` is a Bulma class that renders the checkbox to the right of the label. For some reason this adds a margin-left to the label, which is corrected in the label with `ml-0`. I wanted to use `margin-left-0` to follow the pattern shown here at the top, but Bulma didn't recognize it.

Reply:
I'm guessing "rtl" stands for "Right to Left" (as in for right-to-left languages). Not sure why you want that...?

Reply:
Also, `is-info` is used here for it's color instead of it's semantic meaning. There is one other instance of `is-info` being used for it's color so I figured it's ok.

Reply:
> I'm guessing "rtl" stands for "Right to Left" (as in for right-to-left languages).

I think it means "right of label"?

Reply:
I was following the second example here: https://wikiki.github.io/form/checkradio/

Reply:
Whoops, that does mean right to left... hmm...

Reply:
Or maybe they don't endorse it, but I was confused.

Reply:
It doesn't endorse it, it just gives different examples of use. Also, that's a 3rd party library, not Bulma itself.